docs/resources/Stage_Instance.md

```diff
@@ -103,47 +103,55 @@ Requires the user to be a moderator of the Stage channel.
-| skus                  | array               | skus                                                                                                   |
-
-## List Guild Events % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/events
-
-Returns a list of stage events in the guild.
+### Guild Stage Event Object
+
+###### Guild Stage Event Structure
+
+| Field                 | Type                                                                                                   | Description                                                                                           |
+| --------------------- | ------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------- |
+| id                    | snowflake                                                                                              | the id of the stage event                                                                             |
+| guild_id              | snowflake                                                                                              | the id of the guild the stage event belongs to                                                        |
+| channel_id            | snowflake                                                                                              | the id of the stage channel the stage event belongs to                                                |
+| name                  | string                                                                                                 | the name of the stage event                                                                           |
+| description?          | string                                                                                                 | the description of the stage event                                                                    |
+| image                 | ?string                                                                                                | the image of the stage event                                                                          |
+| scheduled_start_time  | ISO8601                                                                                                | the time the stage event will start                                                                   |
+| scheduled_end_time    | ?ISO8601                                                                                               | the time the stage event will end, or `null` if the stage event does not have a scheduled time to end |
+| privacy_level         | integer                                                                                                | stage event privacy level                                                                             |
```

Comment:
Wouldn't that be [privacy level](#DOCS_RESOURCES_STAGE_INSTANCE/stage-instance-object-privacy-level)

Reply:
Yea, I want to do this, however I don't know the enum names yet. Only thing I know is that the accepted values are `1, 2`

Reply:
Okay

Reply:
actually it may share the same `privacy_level` as Stage Instance now that I think about it, but we maybe might still need more confirmation.

Reply:
Till it's done it can be changed anytime. So for now it should be okay.
As i saw in Datamine, Public Events' aren't available yet anyway.

Reply:
But I assume it'll be more like PUBLIC_EVENT & GUILD_EVENT